<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_receivedMessages">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
